node/actors/consensus/src/replica/state_machine.rs

```diff
@@ -1,10 +1,11 @@
 use crate::{metrics, ConsensusInner};
 use anyhow::Context as _;
-use concurrency::{ctx, metrics::LatencyHistogramExt as _, scope, time};
-use roles::validator;
 use std::collections::{BTreeMap, HashMap};
-use storage::{FallbackReplicaStateStore, StorageError};
 use tracing::instrument;
+use zksync_concurrency::{ctx, metrics::LatencyHistogramExt as _, scope, time};
+use zksync_consensus_roles::validator;
+use zksync_consensus_storage as storage;
```

Comment:
Nit: This is used for `Proposal` and `ReplicaState`, which could arguably be imported directly.

Reply:
I want to keep them prefixed so that it is obvious that these are not from the current crate (and I'm not really fond of renaming the direct imports.